tools/CompileReleaseNotes.py

```diff
@@ -129,6 +131,10 @@ def compile_release_notes(merged_prs: List[PullRequest]) -> str:
                (PULL_REQUEST_GROUPS.index(pr.group), merged_prs.index(pr))),
         key=lambda pr: pr.group)
 
+    # Get set of distinct PR authors. Maintain the order by using a dict, which
+    # is guaranteed to maintain the insertion order since Py 3.7
```

Comment:
is there any chance the release env could use an older Python version and so we should do a sort just in case?

Reply:
The unit tests should fail if the ordering is wrong, I think that suffices.